### ISSUE TITLE
When I see a trail the second time, it doesn't show my "liked" anymore.  

### ISSUE THREAD

**Issue by tgaff** [adamrobertreid/project_03#4]
Users might re-like the same trail multiple times.

**Comment by adamrobertreid** [comment]
Yes, this is a bug I have, I hand some code in my controller which prevented this, it worked once but then it crashed my app. I tried to resolve this issue last night with a TA, to no avail. Currently I have that commented out and is on my to debug list.
